clone-replace.php

```diff
@@ -32,10 +32,35 @@
 add_action(
 	'init',
 	function () {
+		if ( function_exists( 'register_post_meta' ) ) {
+			register_post_meta(
+				'post',
```

Comment:
Does this plugin work with custom post types?  Should this be for all post types that are supported?  Same question for the next register_post_meta call.

Reply:
The current implementation for the classic editor has no support for other post types so I decided to follow up here.